.github/workflows/nightly.yml

```diff
@@ -171,6 +171,7 @@ jobs:
           sonar-scanner \
             --define sonar.host.url="${{ env.SONAR_SERVER_URL }}" \
             --define sonar.cfamily.build-wrapper-output="${{ env.BUILD_WRAPPER_OUT_DIR }}" \
+            --define sonar.cfamily.reportingCppStandardOverride=c++20 \
```

Comment:
Maybe add a TODO comment to specify C++23 once Sonar supports it (ClickHouse compiles with C+23 since a while).